src/Symfony/Component/Config/Resource/FileResource.php

```diff
@@ -44,7 +44,7 @@ public function __toString()
     }
 
     /**
-     * {@inheritdoc}
+     * @return string The file path to the resource
```

Comment:
currently it can also return `false` when realpath returns an error. So I propose to throw an exception in the constructor in this case which is ok in 3.0 i guess.